source/manual/reindex-elasticsearch.html.md.erb

```diff
@@ -7,139 +7,110 @@ parent: "/manual.html"
 related_repos: [search-api]
 ---
 
-After updating an Elasticsearch index's schema by [changing the fields or
-document types][update-fields-or-doc-types], you need to reindex the affected
-index before the new fields and types can be used.
+## When to reindex
 
-The reindexing process:
+After updating an Elasticsearch index's schema by [changing the fields or
+document
+types](https://github.com/alphagov/search-api/blob/main/docs/adding-new-fields.md),
+you need to reindex the affected index before the new fields and types can be
+used.
+
+> ⚠️ **Avoid reindexing in production during working hours, except to fix an
+> outage.**
+>
+> Reindexing locks the index for writes, so content is not updated in the
+> search index. Reindexing takes around 2 hours to complete.
```

Comment:
```suggestion
> search index. Reindexing takes around 2 hours to complete.
```

If someone can confirm that writes are queued and retried (and therefore indexing should eventually catch up on its own after the reindex finishes) then I'd love to clarify that here.

I'd expect that's the case but I didn't want to assume (and I don't have time to verify it experimentally).